cardillo/math/rotations.py

```diff
@@ -565,9 +568,9 @@ def Exp_SO3_quat(P, normalize=True):
         return (p0**2 - p @ p) * eye3 + np.outer(p, 2 * p) + 2 * p0 * ax2skew(p)
```

Comment:
I would suggest to return `return eye3 + 2 * (p0 * ax2skew(p) + ax2skew_squared(p))` in the `else` case. Hence, only the factor `1 / P2` changes.